src/config/dynamic-theme-fixes.config

```diff
@@ -484,6 +484,21 @@ INVERT
 
 ================================
 
+messages.google.com
+
+INVERT
+.x4Tquc
+.QrWqSe
+.XCHXxd
+.pXeIKc
+
+CSS
+.input-background {
+  --input-bg-fade-color: #202124 !important;
```

Comment:
Please note, that color should look well in light mode too, so `${Default_Color}` template is used to apply Filter values.